src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Represents a field of a tuple type (such as (int, byte).Item1)
+    /// that doesn't have a corresponding backing field within the tuple underlying type.
+    /// Created in response to an error condition.
+    /// </summary>
+    internal sealed class TupleErrorFieldSymbol : SynthesizedFieldSymbolBase
+    {
+        private readonly TypeSymbol _type;
+
+        /// <summary>
+        /// If this field represents a tuple element (including the name match), 
+        /// id is an index of the element (zero-based).
+        /// Otherwise, (-1 - [index in members array]);
+        /// </summary>
+        private readonly int _tupleFieldId;
+
+        private readonly ImmutableArray<Location> _locations;
+        private readonly DiagnosticInfo _useSiteDiagnosticInfo;
+
+        public TupleErrorFieldSymbol(NamedTypeSymbol container, string name, int tupleFieldId, Location location, TypeSymbol type, DiagnosticInfo useSiteDiagnosticInfo)
+            : base(container, name, isPublic:true, isReadOnly:false, isStatic:false)
+        {
+            Debug.Assert(name != null);
+            _type = type;
+            _locations = location == null ? ImmutableArray<Location>.Empty : ImmutableArray.Create(location);
+            _useSiteDiagnosticInfo = useSiteDiagnosticInfo;
+            _tupleFieldId = tupleFieldId;
+        }
+
+        /// <summary>
+        /// If this field represents a tuple element (including the name match), 
+        /// id is an index of the element (zero-based).
+        /// Otherwise, (-1 - [index in members array]);
+        /// </summary>
+        public override bool IsTupleField
```

Comment:
Correct, will fix in the next change.

---
In reply to: [61812024](https://github.com/dotnet/roslyn/pull/11006#discussion_r61812024) [](ancestors = 61812024)